server/protocols/webadmin-integration-test/memory-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/memory/MemoryWebAdminServerIntegrationTest.java

```diff
@@ -39,4 +51,32 @@ class MemoryWebAdminServerIntegrationTest extends WebAdminServerIntegrationTest
             .build())
         .server(MemoryJamesServerMain::createServer)
         .build();
+
+    @RegisterExtension
+    TestIMAPClient testIMAPClient = new TestIMAPClient();
+
+    @Test
+    void shouldDescribeConnectedImapChannels(GuiceJamesServer server) throws Exception {
+        int imapPort = server.getProbe(ImapGuiceProbe.class).getImapPort();
+
+        server.getProbe(DataProbeImpl.class).addUser(USERNAME, PASSWORD);
+
+        testIMAPClient.connect(LOCALHOST_IP, imapPort);
+        testIMAPClient.sendCommand("ID (\"name\" \"Thunderbird\" \"version\" \"102.7.1\")");
```

Comment:
```suggestion
        testIMAPClient.sendCommand(""" ID ("name"  "Thunderbird"  "version"  "102.7.1") """);
```

Reply:
![image](https://github.com/user-attachments/assets/302480fc-44f4-494b-a2be-dfaad051fbb3)

TLDR choose your poinson" I'll keep the espace sequences.